spec/features/admin/enrollments/edit_spec.rb

```diff
@@ -5,40 +5,60 @@
     create(:enrollment, location: 'Eloise May')
   end
 
-  before do
-    visit edit_admin_enrollment_path(eloise_may)
+  let(:date_field_css) do
+    'input[type="text"][class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"][placeholder="MM/DD/YYYY"]'
   end
 
-  it "displays a form pre-filled with the enrollment's data" do
-    expect(page).to have_content('Please fill the form below:')
-    expect(page).to have_selector(:css, 'form')
-    expect(page).to have_field('student_limit', with: '30')
-    expect(page).to have_select('location', selected: 'Eloise May')
-    expect(find('input[type="text"][class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"][placeholder="MM/DD/YYYY"]').value).to eq("#{eloise_may.schedule.in_time_zone('Mountain Time (US & Canada)').strftime('%m/%d/%Y')}")
-    expect(find('input[type="text"][class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"][placeholder="hh:mm aa"]').value).to eq("#{eloise_may.schedule.in_time_zone('Mountain Time (US & Canada)').strftime('%I:%M %p')}")
-    expect(page).to have_button('Submit')
+  let(:time_field_css) do
```

Comment:
Same as above but I love it so much that here you go again:

<img src="https://media2.giphy.com/media/JmCFfTlNy12wFpOK8Y/giphy.gif"/>